OWNERS_ALIASES

```diff
@@ -24,3 +24,4 @@ aliases:
     - Jont828
     - jsturtevant
     - sonasingh46
+    - marosset
```

Comment:
nit: list should be alphabetically ordered

Reply:
done!